ui/components/component-library/component-library-components.scss

```diff
@@ -1,3 +1,4 @@
 /** Please import your files in alphabetical order **/
 @import 'base-avatar/base-avatar';
 @import 'base-icon/base-icon';
+@import 'text/text';
```

Comment:
You may need a new line break after this line

```suggestion
@import 'text/text';

```

<img width="658" alt="Screen Shot 2022-08-18 at 8 14 08 AM" src="https://user-images.githubusercontent.com/8112138/185433065-516d9401-1f68-4a19-8a6d-923170364058.png">